my_game.py

```diff
@@ -579,11 +589,16 @@ def on_update(self, delta_time):
         for s in self.player_shot_list:
 
             for a in arcade.check_for_collision_with_list(s, self.asteroid_list):
+                for n in range(ASTEROIDS_PR_SPLIT):
+                    if a.size > 1:
+                        self.asteroid_list.append(Asteroid(a.size-1, a.center_x, a.center_y, random.randrange(a.direction-30, a.direction+30)))
+                    else:
+                        pass
                 s.kill()
                 a.kill()
                 self.sound_explosion.play()
-                self.player_score += ASTEROID_POINT_VALUE
-
+                self.player_score += ASTEROID_SCORE_VALUES[a.size-1]
```

Comment:
Maybe we could store the value in the Asteroid itself, so you could say:
```
self.player_score += a.value
```

What do you think?

Reply:
that would of course make it more flexible for change in the future. ill do it